src/sql/workbench/contrib/charts/browser/chartView.ts

```diff
@@ -68,7 +69,7 @@ export class ChartView extends Disposable implements IPanelView {
 	/** parent container */
 	private container: HTMLElement;
 	/** container for the options controls */
-	private optionsControl: HTMLElement;
+	public readonly optionsControl: HTMLElement;
```

Comment:
It seems like with your changes it would be good to pull the options out into its own class so we can have more direct control over it since it's not always going to be directly tied to the chart UI-layout-wise

Reply:
Currently the dialog is just an alternate display method. Rendering just the optionsControl element elsewhere seemed to be a less disruptive way of achieving that.